src/Plugin.php

```diff
@@ -203,6 +203,14 @@ public static function init() {
 
     // Disables output of related products if over-ride checkbox is enabled.
     add_action('woocommerce_after_single_product_summary', __NAMESPACE__ . '\WooCommerce::disableRelatedProducts', 0, 2);
+
+    /**
+     * Removes the `/page/1` suffix from archive URLs.
+     */
```

Comment:
Only a inline comment (single line summary) here; PHPDoc block is only for the separate function definition.